stl/inc/yvals_core.h

```diff
@@ -855,6 +855,42 @@
 
 // next warning number: STL4027
 
+// P0619R4 remove deprecated features from C++20
+#ifndef _HAS_REMOVED_FEATURES_CXX20
+#define _HAS_REMOVED_FEATURES_CXX20 (!_HAS_CXX20)
+#endif // _HAS_REMOVED_FEATURES_CXX20
+
+#ifndef _HAS_DEPRECATED_TYPETRAITS

```

Comment:
Alternatively, I think we should probably have separate escape hatches for `RESULT_OF` and `IS_LITERAL_TYPE`, mirroring the separate deprecation silencing macros. This is because `result_of` will be restored 1000x more than `is_literal_type`.

Reply:
I split them up into `_HAS_DEPRECATED_IS_LITERAL_TYPE` and `_HAS_DEPRECATED_RESULT_OF`